internal/app/rds/rds_test.go

```diff
@@ -94,6 +94,7 @@ func newRdsInstance() *aws_rds_types.DBInstance {
 		CACertificateIdentifier:    aws.String("rds-ca-2019"),
 		CertificateDetails:         newRdsCertificateDetails(),
 		InstanceCreateTime:         &now,
+		TagList:                    []aws_rds_types.Tag{{Key: aws.String("Environment"), Value: aws.String("unittest")}, {Key: aws.String("Team"), Value: aws.String("sre")}},
```

Comment:
```suggestion
		Tags:                    []aws_rds_types.Tag{{Key: aws.String("Environment"), Value: aws.String("unittest")}, {Key: aws.String("Team"), Value: aws.String("sre")}},
```

Reply:
It's an AWS `DBInstance` structure. I can't change the name. But I named it `Tags` in our codebase.

Reply:
Darn it, you AWS people 😡😂  Sorry I read that too quickly!